common/app/model/Cached.scala

```diff
@@ -49,7 +49,7 @@ object Cached extends implicits.Dates {
 }
 
 object NoCache {
-  def apply(result: Result): Result = result.withHeaders("Cache-Control" -> "no-cache", "Pragma" -> "no-cache")
+  def apply(result: Result): Result = result.withHeaders("Cache-Control" -> "no-cache, no-store, must-revalidate, max-age=0", "Pragma" -> "no-cache")
```

Comment:
@rich-nguyen are you sure it's right to add that or are you just trying to throw everything at it? `the Expires header is used to specify a specific point in time the resource is no longer valid`